Packs/Polygon/Playbooks/playbook-Detonate_File_-_Group-IB_TDS_Polygon.yml

```diff
@@ -371,23 +436,125 @@ tasks:
           transformers:
           - operator: LastArrayElement
     separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1030,
+          "y": 1220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 2cec22a6-b48e-474b-84ee-4aec0f057084
+    type: regular
+    task:
+      id: 2cec22a6-b48e-474b-84ee-4aec0f057084
+      version: -1
+      name: 'Set report number to content '
```

Comment:
Check if we need to use this task of `set` - if not pls update the PB + update the image